sql/functions/functions.sql

```diff
@@ -628,3 +628,107 @@ END IF;
 
 END
 $function$;
+
+CREATE FUNCTION @extschema@.ccp_stat_bgwriter() RETURNS TABLE
+(
+    buffers_clean bigint
+    , maxwritten_clean bigint
+    , buffers_alloc bigint
+)
+    LANGUAGE plpgsql
+AS $function$
+DECLARE
+BEGIN
+
+IF current_setting('server_version_num')::int >= 170000 THEN
+
+    RETURN QUERY
+    SELECT
+        b.buffers_clean
+        , b.maxwritten_clean
+        , b.buffers_alloc
+    FROM pg_catalog.pg_stat_bgwriter b;
+
+ELSE
+
+    RETURN QUERY
+    SELECT
+        b.buffers_clean
+        , b.maxwritten_clean
+        , b.buffers_alloc
+    FROM pg_catalog.pg_stat_bgwriter b;
```

Comment:
Are these supposed to be the same query?

Reply:
Hah! This was left over from my initial writeup of this where I only had one metric for everything. I'd slowly moved the pieces from this one to the other new ones and of course it left these equivalent in the end.

Reply:
Thanks for that catch. Removed the function for that and just made it a normal view